sdk/test/UnitTests/Custom/Runtime/Credentials/SharedCredentialsFileTest.cs

```diff
@@ -497,7 +512,31 @@ public void ReadProfileWithSubproperties()
                     {"name","value" }
                 };
                 expectedNestedProperties.Add("s3", entry);
-                tester.ReadAndAssertProfile("foo", BasicProfileOptions,  expectedNestedProperties);
+                tester.ReadAndAssertProfile("foo", BasicProfileOptions, expectedNestedProperties);
+            }
+        }
+
+        [TestMethod]
+        public void ReadProfileWithSubpropertiesPropertyKeyWithMixedCase()
+        {
+            using (var tester = new SharedCredentialsFileTestFixture(CredentialsProfilePropertyKeyWithMixedCase, ConfigProfileWithSubpropertiesPropertyKeyWithMixedCase))
+            {
+                CredentialProfileOptions expectedProfileOptions = new CredentialProfileOptions
+                {
+                    AccessKey = "basic_aws_ACCESS_key_id",
+                    SecretKey = "basic_Aws_sEcRet_AcceSS_key",
+                };
+                Dictionary<string, string> expectedProperties = new Dictionary<string, string>
+                {
+                    { "output", "json" }
+                };
+                Dictionary<string, Dictionary<string, string>> expectedNestedProperties = new Dictionary<string, Dictionary<string, string>>();
```

Comment:
Nit, but `Dictionary<string, Dictionary<string, string>> expectedNestedProperties = new Dictionary<string, Dictionary<string, string>>();` is really hard to read. 

I'd use `var expectedNestedProperties = new ...` instead (same comment applies to all variables in this test).

Reply:
Done